x-pack/plugins/security_solution/server/lib/entity_analytics/risk_score/calculate_risk_scores.ts

```diff
@@ -305,8 +315,20 @@ export const calculateRiskScores = async ({
       index,
       runtime_mappings: runtimeMappings,
       query: {
-        bool: {
-          filter,
+        function_score: {
+          query: {
+            bool: {
+              filter,
+              should: [
+                {
+                  match_all: {}, // This forces ES to calculate score
+                },
+              ],
+            },
+          },
+          field_value_factor: {
+            field: 'kibana.alert.risk_score', // sort by risk score
```

Comment:
This string is referenced elsewhere in this file as `ALERT_RISK_SCORE`

Reply:
Will be resolved by https://github.com/elastic/kibana/pull/179331